src/aap_eda/tasks/ruleset.py

```diff
@@ -29,7 +30,7 @@
 
 
 @job("activation")
-def activate_rulesets(is_restart: bool, activation_id: int) -> None:
+def activate(activation_id: int) -> None:
```

Comment:
Activation can be implicit as a result of other actions (e.g., create), but wouldn't it be reasonable to have this be consistent with the other methods in having a `requester` parameter?

Reply:
@mkanoor I figured as much.  It's just that it seems odd to have what appear to be similar actions with different interfaces.  I'd be comfortable enough if it was commented as to why it's different.

Reply:
@jshimkus-rh You are correct, we could have activation either started by the User using the UI or the scheduler task when it sees that the Activation is non responsive. The requester is used in log messages to see who started the process was it a scheduler or the user. @hsong-rh we should add a requester field in this function so the log messages are clearer about who initiated the process (activate/disable/delete). I think delete is the only task that is done by the end user, but the restart and disable could be done by both the scheduler and the user.

Reply:
@jshimkus-rh @mkanoor the `requester` is added to `activate` too